src/app/shared/toolbar/toolbar.component.html

```diff
@@ -1,75 +1,42 @@
-<mat-toolbar class="fixed-toolbar" color="primary">
-
-
-
-  <div fxHide fxShow.gt-sm class="toolbar-content">
-
-    <a class="store-logo-button" routerLink="/" aria-label="Flathub">
-      <img class="store-logo" src="../../../assets/themes/flathub/flathub-logo-toolbar.svg" alt="flathub">
-    </a>
-
-    <div><a mat-button routerLink="apps">APPLICATIONS</a></div>
-
-
-    <span class="flex-fill-remaining-space"></span>
-
-
-    <div class="search-box-container">
-      <mat-icon class="search-box-icon">search</mat-icon>
-      <input #searchBox id="search-box" class="search-box" type="search" placeholder="Search apps" (keyup.enter)="onSearch(searchBox.value)"
-      />
-    </div>
-
+<mat-toolbar color="primary">
+  <div class="toolbar-content">
     <div>
-    <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">
-      PUBLISH
-    </a>
-    <a mat-button routerLink="about">ABOUT</a>
-  </div>
-  </div>
-
-  <div fxShow fxHide.gt-sm class="toolbar-content">
+      <a class="store-logo-button" routerLink="/" aria-label="Flathub" mat-button>
+        <img src="../../../assets/themes/flathub/flathub-logo-toolbar.svg" alt="flathub">
+      </a>
+    </div>
 
-    <button *ngIf="!showSearchInput" mat-icon-button [matMenuTriggerFor]="menu" xPosition="after">
-      <mat-icon>more_vert</mat-icon>
-    </button>
+    <div class="flex-fill-remaining-space">
+      <div class="search-box-container">
+        <div [class]="'search-form '+(searchActive? 'active': '')">
+          <mat-icon class="search-box-icon show-md" (click)="searchActive=!searchActive">arrow_back</mat-icon>
+          <mat-icon class="search-box-icon hide-md">search</mat-icon>
+          <input #searchBox id="search-box" class="search-box" type="search" placeholder="Search apps" (keyup.enter)="onSearch(searchBox.value)" />
+        </div>
+      </div>
+    </div>
 
-    <mat-menu #menu="matMenu">
+    <div class="hide-md toolbar-nav">
+      <button mat-button routerLink="apps">APPLICATIONS</button>
+      <button mat-button href="https://github.com/flathub/flathub/wiki/App-Submission">PUBLISH</button>
```

Comment:
External links don't work with button. The code should be like this:

` <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">PUBLISH</a>`